pkg/virt-launcher/virtwrap/network/podinterface.go

```diff
@@ -45,6 +45,8 @@ import (
 
 var bridgeFakeIP = "169.254.75.1%d/32"
 
+const hardcodedMasqueradeMAC = "02:82:cf:9f:09:83"
```

Comment:
not on the same prefix as the generated mac addresses for the vms on masquerade bindings.